gamemode/modules/fadmin/fadmin/playeractions/kickban/sv_init.lua

```diff
@@ -144,19 +144,19 @@ local function Ban(ply, cmd, args)
     local time
 
     for _, target in pairs(targets) do
-        if (type(target) == "string" and not FAdmin.Access.PlayerHasPrivilege(ply, "Ban")) or
+        if (isstring(target) and not FAdmin.Access.PlayerHasPrivilege(ply, "Ban")) or
```

Comment:
The isstring checks here and below are kinda duplicitous, try to rework the if-statement structure to only perform it once.

Reply:
I don't understand the problem.

Reply:
There are other isstring checks of the same type here in the same loop
https://github.com/FPtje/DarkRP/blob/a444f882d4f9f8e7a968f35c499bd7b1d1699ea2/gamemode/modules/fadmin/fadmin/playeractions/kickban/sv_init.lua#L152 https://github.com/FPtje/DarkRP/blob/a444f882d4f9f8e7a968f35c499bd7b1d1699ea2/gamemode/modules/fadmin/fadmin/playeractions/kickban/sv_init.lua#L159 https://github.com/FPtje/DarkRP/blob/a444f882d4f9f8e7a968f35c499bd7b1d1699ea2/gamemode/modules/fadmin/fadmin/playeractions/kickban/sv_init.lua#L159 https://github.com/FPtje/DarkRP/blob/a444f882d4f9f8e7a968f35c499bd7b1d1699ea2/gamemode/modules/fadmin/fadmin/playeractions/kickban/sv_init.lua#L168 https://github.com/FPtje/DarkRP/blob/a444f882d4f9f8e7a968f35c499bd7b1d1699ea2/gamemode/modules/fadmin/fadmin/playeractions/kickban/sv_init.lua#L168 https://github.com/FPtje/DarkRP/blob/a444f882d4f9f8e7a968f35c499bd7b1d1699ea2/gamemode/modules/fadmin/fadmin/playeractions/kickban/sv_init.lua#L185

He wants there to be only one check instead of all this.

Reply:
I think it's a topic for a separate PR.